CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ CHANGELOG
 ## HEAD (Unreleased)
 * Add several properties to ArchiveFunctionApp.
   ([#514](https://github.com/pulumi/pulumi-azure/pull/514))
+* Upgrade to v2.5.0 of the AzureRM Terraform Provider
+  ** PLEASE NOTE:** 
+  Unfortunately, there are some breaking changes inrespect to 
```

Comment:
Perhaps remove “unfortunately” and “inrespect”?

Reply:
done in the changelog